internal/fs/fs_reader_test.go

```diff
@@ -317,6 +327,49 @@ func TestFSReader(t *testing.T) {
 			test.f(t, fs)
 		})
 	}
+
+	var dirTests = []struct {
```

Comment:
I'd split this out into its own test function, maybe `TestFSReaderDir`?

Reply:
yes, that's clearer.